topgun/k8s/baggageclaim_drivers_test.go

```diff
@@ -36,35 +36,18 @@ var _ = Describe("Baggageclaim Drivers", func() {
 	DescribeTable("across different node images",
 		func(c Case) {
 			releaseName = fmt.Sprintf("topgun-bd-%s-%s-%d-%d",
-				c.Driver, c.NodeImage, GinkgoRandomSeed(), GinkgoParallelNode())
+				c.Driver, c.NodeImage, randomGenerator.Int(), GinkgoParallelNode())
```

Comment:
Oh, so the problem was that we were trying to delete a namespace that had a naming collision with something else?
I wonder if we should then just drop `GinkgoParallelNode()` to as the call to `.Int()` is going to give us a different number each time we call anyway. 

Wdyt?

thx!

Reply:
ok, I am also thinking about getting a smaller number ... this one is making the `releasename` huge.